src/cargo-fmt/main.rs

```diff
@@ -47,6 +47,9 @@ pub struct Opts {
     message_format: Option<String>,
 
     /// Options passed to rustfmt
+    ///
+    /// To see all rustfmt options, run `rustfmt --help' or visit
+    /// https://rust-lang.github.io/rustfmt.
```

Comment:
Is this url really valid? :thinking: That'll get users to config options but not the full help text with flags/opts/etc.

Reply:
good catch, I updated the text accordingly